frontend/src/generate-static-html.js

```diff
@@ -1,24 +1,42 @@
 
```

Comment:
This is fixed now, generate-static-html.js gets put in build/ instead of build/static/app/ and does not get shipped to the site.